client/rpcserver/types.go

```diff
@@ -11,3 +13,31 @@ type VersionResult struct {
 	Minor uint32 `json:"minor"`
 	Patch uint32 `json:"patch"`
 }
+
+// ParseCmdArgs parses arguments to commands for rpcserver requests.
+func ParseCmdArgs(cmd string, args []interface{}) (interface{}, error) {
+	switch cmd {
+	case "help":
+		return parseHelpArgs(args)
+	case "version":
+		return parseVersionArgs(args)
```

Comment:
How about a `map[string]func([]interface{}) (interface{}, error)`?

Reply:
oh, that's great. thank you